sklearn/preprocessing/data.py

```diff
@@ -250,9 +260,22 @@ def fit(self, X, y=None):
         if feature_range[0] >= feature_range[1]:
             raise ValueError("Minimum of desired feature range must be smaller"
                              " than maximum. Got %s." % str(feature_range))
-        data_min = np.min(X, axis=0)
-        data_range = np.max(X, axis=0) - data_min
+
+        if len(X.shape) == 2:
```

Comment:
Yes I should. (I was considering adding a warning, at least.) I'll put that in a future commit, thank you.